.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__
 .vscode
-env/
\ No newline at end of file
+env/
+.idea/*
```

Comment:
What's this for?

Reply:
I use Jetbrains (Pycharm) as my IDE and it creates session variables and so on. Those variables are stored in this folder.